src/app.css

```diff
@@ -176,3 +176,41 @@ body, html {
 .icon-slash:before {
     content: "\e90d";
 }
+.bg-red-500 {
+    --tw-bg-opacity: 1;
+    background-color: rgb(185 16 16/ var(--tw-bg-opacity));
+}
+.hover\:bg-red-400:hover {
+    --tw-bg-opacity: 1;
+    background-color: rgb(211 16 16 / var(--tw-bg-opacity));
+}
+.bottomBar {
+    grid-template-columns: 1fr 4fr 1fr;
+}
+.addressInput {
+    width: 20vw;
+}
+.passwordField {
+    width: 220px;
+}
```

Comment:
Just ignore these, I cannot choose what lines I want to commit, so yeah just deal with it, they do nothing, but if I don't leave them there I get issues with stupid git not wanting to pull/push